src/components/Display.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { value } = props;
```

Comment:
### [Optional]
To make the code more clear, you might wanna rename `value` to `result`
![image](https://user-images.githubusercontent.com/31897434/79173554-d4db0280-7def-11ea-9938-e36c89e019f5.png)